package.json

```diff
@@ -7,6 +7,7 @@
     "firebase": "^9.1.1",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
+    "react-icons": "^4.3.1",
```

Comment:
Nice! I didn't know about it. Thank you for using those icons ⭐ !

Reply:
Small details 😄 : I think it could be helpful please if you can update your PR description, and check ✔️ "Update dependencies" and "Testing steps" for the next mentors and teammates.

Reply:
Thank you Alejandro! I will update this!